llvm-general/src/LLVM/General/Internal/FFI/TargetC.cpp

```diff
@@ -338,22 +316,4 @@ void LLVM_General_InitializeAllTargets() {
   // None of the other components are bound yet
 }
 
-LLVMBool LLVM_General_TargetMachineEmit(
```

Comment:
What's your motivation for removing this function and changing the factoring of the various emitting functions in Module.hs?